resources/views/emails/messages/plain_text.blade.php

```diff
@@ -0,0 +1,6 @@
+Dear {!! isset($recipientName) ? $recipientName : $recipient->name !!},
+
+{{  strip_tags($body) }}
```

Comment:
`htmlspecialchars` will replace the html special characters with their `&___;` counterpart, so it is nesseary to completely remove these before that is run.